.github/bump-formula.rb

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+ARCHS = %w[darwin_amd64 darwin_arm64 linux_amd64 linux_arm linux_arm64].freeze
+
+OLD_VERSION = ARGV[0]
+NEW_VERSION = ARGV[1]
+
+Dir.entries("#{File.dirname(__FILE__)}/Formula/").each do |file|
+  next unless file.end_with?('.rb')
+
+  puts "Processing #{file}"
+
+  refs = {}
+  formula = file.chomp('.rb')
+
+  ARCHS.each do |arch|
+    url = "https://cli.upbound.io/stable/#{OLD_VERSION}/bundle/#{formula}/#{arch}.tar.gz"
+    old_sha256 = `curl -sSL #{url} | sha256sum | cut -d' ' -f1`.strip
+
+    url = "https://cli.upbound.io/stable/#{NEW_VERSION}/bundle/#{formula}/#{arch}.tar.gz"
+    new_sha256 = `curl -sSL #{url} | sha256sum | cut -d' ' -f1`.strip
+
+    refs[arch] = {
+      old_sha256: old_sha256,
+      new_sha256: new_sha256
+    }
+  end
+
+  text = File.read("#{File.dirname(__FILE__)}/Formula/" + file)
```

Comment:
Will this work in the .github directory?

Reply:
I think the action will execute from the root directory, so it should?

Reply:
Ah nice 👍🏻 didn't realize action invoked using full relative path

Reply:
You were right 🤦‍♂️ : https://github.com/upbound/homebrew-tap/pull/26